lib/rules/comma-dangle.js

```diff
@@ -128,10 +132,14 @@ module.exports = function(context) {
         var sourceCode = context.getSourceCode();
         var trailingToken = sourceCode.getTokenAfter(lastItem);
         if (trailingToken.value !== ",") {
-            context.report(
-                lastItem,
-                lastItem.loc.end,
-                MISSING_MESSAGE);
+            context.report({
+                node: lastItem,
+                loc: lastItem.loc.end,
+                message: MISSING_MESSAGE,
+                fix: function(fixer) {
+                    return fixer.insertTextAfter(lastItem, ",");
```

Comment:
This could cause the `comma-spacing` rule to report errors (depends on how the rule is configured). I’m not sure how we should deal with such situations and if we should aim to avoid auto fixes from causing other rules to report errors.